src/sentry/runner/commands/run.py

```diff
@@ -572,3 +572,27 @@ def profiles_consumer(**options):
     from sentry.profiles.consumer import get_profiles_consumer
 
     get_profiles_consumer(**options).run()
+
+
+@run.command("last-seen-updater")
+@log_options()
+@configuration
+@batching_kafka_options("snuba-metrics-consumer")
```

Comment:
The group has to be different than the default we use in Snuba.
Otherwise you will try to join the same cluster of consumers.
Consumer groups allow you to consume the same topic independently. But if you reuse the name, you are going to take partitions away from the existing consumers clusters, so you are not going to consume the same messages independently.